### PR TITLE
model update ff projects

### PR DESCRIPTION
**Pull request by erichesse**
# PR Context
- model update
- [out of scope]  applied requested changes to meta data extraction(metadata review pending)

### CODE REVIEW

File: mex/extractors/ff_projects/extract.py [hunk 1]
```diff
@@ -19,6 +19,21 @@
...
+    "BBK": "Bundesamt für Bevölkerungsschutz und Katastrophenhilfe",
+    "DGE": "Deutsche Gesellschaft für Ernährung",
+    "FLI": "Friedrich Loeffler Institute",
+    "BMWK (ehem. BMWi)": "Federal Ministry for Economic Affairs and Climate Action",
+    "BMWK": "Federal Ministry for Economic Affairs and Climate Action",
+    "BMWiW": "Federal Ministry for Economic Affairs and Climate Action",
+    "IBB": "Investitionsbank Berlin",
+    "Sepsis-Stiftung": "Sepsis-Stiftung",
+    "DG-Sante": "Directorate-General for Health and Food Safety",
+}
```
Comment (cutoffthetop) [inline mex/extractors/ff_projects/extract.py:35]: can't we read this from the mapping?
i know, currently it's not quite machine-friendly, but we could just put the values to use into `setValues`, no?

**Review by cutoffthetop — CHANGES_REQUESTED** [review]
looks good, just one question

.. and i have 2 failing tests, do they work on your machine?

```
FAILED tests/ff_projects/test_main.py::test_job - mex.common.exceptions.MExError: MExError: missing unit should have been filtered out
FAILED tests/ff_projects/test_filter.py::test_filter_and_log_ff_projects_sources - assert 15 == 2
```

**Comment by erichesse** [comment]
> looks good, just one question
> 
> .. and i have 2 failing tests, do they work on your machine?
> 
> ```
> FAILED tests/ff_projects/test_main.py::test_job - mex.common.exceptions.MExError: MExError: missing unit should have been filtered out
> FAILED tests/ff_projects/test_filter.py::test_filter_and_log_ff_projects_sources - assert 15 == 2
> ```

The filtering is a weird issue itself, I don't really find it in the mapping. I think we need to for the metadata check...

File: mex/extractors/ff_projects/extract.py (hunk 1 above)
Comment (erichesse) [reply mex/extractors/ff_projects/extract.py:35]: I made a remark in the ticket, to change the mapping accordingly.

**Comment by erichesse** [comment]
> > looks good, just one question
> > .. and i have 2 failing tests, do they work on your machine?
> > ```
> > FAILED tests/ff_projects/test_main.py::test_job - mex.common.exceptions.MExError: MExError: missing unit should have been filtered out
> > FAILED tests/ff_projects/test_filter.py::test_filter_and_log_ff_projects_sources - assert 15 == 2
> > ```
> 
> The filtering is a weird issue itself, I don't really find it in the mapping. I think we need to for the metadata check...

I reverted the changes that broke the tests and created MX-1671 for wrap up. I think we should merge model update like this now,

**Review by cutoffthetop — APPROVED** [review]

**Pull request merged**